docs/develop/get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md

```diff
@@ -27,11 +27,11 @@ This article assumes that you are using a text editor to create the task pane ad
 Alternatively, you can create an add-in for your Office application using one of the following options. You won't have to create new files as the equivalent of each required file will be available for you to update. For example, the Yeoman generator options include **./src/taskpane/taskpane.html**, **./src/taskpane/taskpane.js**, **./src/taskpane/taskpane.css**, and **./manifest.xml**.
 
 - PowerPoint
-  - [Visual Studio](../quickstarts/powerpoint-quickstart.md?tabs=visualstudio)
-  - [Yeoman generator for Office Add-ins](../quickstarts/powerpoint-quickstart.md?tabs=yeomangenerator)
+  - [Visual Studio](../quickstarts/powerpoint-quickstart-vs.md)
+  - [Yeoman generator for Office Add-ins](../quickstarts/powerpoint-quickstart-yo.md)
```

Comment:
I understand the thought process. My logic is that if we add excel-quick-start-x.md, only that file and the TOC will be edited. I'll think about it and welcome other opinions.

Reply:
@AlexJerabek in the event that there's a new default, does that mean we need to keep updating the redirects too?

Reply:
@ElizabethSamuel-MSFT, No, because in that case, the -yo files remain, so links to them will still be valid. I think that's a benefit because the external link might say something like "To use a yeoman generator for Office Add-ins, see this quick start".